core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -622,8 +626,12 @@ def onComplete(self):
         self.logger.info('Validation of file complete')
 
     def processTopLines(self, line_list):
-        """Overide to parse any list of comment lines above the TSV header."""
-        pass
+        """Hook to parse any list of comment lines above the TSV header.
+
```

Comment:
same comment here. This is not something I would expect in the generic Validator class, but in ClinicalValidator